src/Components/Map.jsx

```diff
@@ -64,11 +64,7 @@ class Map extends Component {
       >
 
         <Popup modal trigger={
-          <Icon style={{
-            position: 'absolute',
-            zIndex: '4000',
-            padding: '1rem'
-          }}
+          <Icon
```

Comment:
Just out of interest: are these styling attributes not needed or unnecessary or set in the site overrides instead? To understand why you took them away!

Reply:
All this time we had them both in the overrides and here :P